setup.cfg

```diff
@@ -1,6 +1,2 @@
 [flake8]
 max-line-length = 88
-exclude = \
-    build, \
-    setup.py, \
-    docs/source/conf.py
```

Comment:
If I run `flake8` with this locally I get
```
./docs/source/conf.py:369:1: E266 too many leading '#' for block comment
./docs/source/conf.py:378:1: E266 too many leading '#' for block comment
./docs/source/conf.py:392:1: E266 too many leading '#' for block comment
```
Is it worth reinstating this exclude?

Reply:
Yeah, good catch, I should have just left that file alone. Just fixed that in the latest push.